engine/rotationmanager/legacyadvance.go

```diff
@@ -0,0 +1,53 @@
+package rotationmanager
+
+import (
+	"time"
+
+	"github.com/target/goalert/schedule/rotation"
+)
+
+func addHours(t time.Time, n int) time.Time {
+	return time.Date(t.Year(), t.Month(), t.Day(), t.Hour()+n, t.Minute(), t.Second(), t.Nanosecond(), t.Location())
+}
+
+func addHoursAlwaysInc(t time.Time, n int) time.Time {
```

Comment:
maybe `addHoursAbs` would be more appropriate? that way a dev would know it would do `|n|` on neg, rather than say `0` or `1`

```suggestion
func addHoursAbs(t time.Time, n int) time.Time {
```

Reply:
It was named because it guarantees time will always increment, but I'm going to prefix them with `legacy` and comment as such. These aren't meant to be used going forward -- they were copied 1-to-1 from old code for compatibility during the migration only.